recipes/graphvite/conda_build_config.yaml

```diff
@@ -0,0 +1,18 @@
+cxx_compiler_version:
+  - 5.4
+
+python:
+#  - 2.7
+  - 3.6
+  - 3.7
+
+numpy:
+  - 1.11
+
+cudatoolkit:
+  - 9.2
+  - 10.0
+
+pin_run_as_build:
+  cudatoolkit:
+    max_pin: x.x
```

Comment:
This file can be dropped. We already use newer C++ compilers. Also Python, NumPy, and CUDA are already configured globally.